sources/engine/Stride.Graphics/CommandList.cs

```diff
@@ -20,11 +20,10 @@ public partial class CommandList : GraphicsResourceBase
 
         private int boundScissorCount;
         private readonly Rectangle[] scissors = new Rectangle[MaxViewportAndScissorRectangleCount];
-        private bool scissorsDirty = false;
```

Comment:
This is set multiple places but never used for anything.

Reply:
It's read in D3D12 and Vulkan CommandLists, please revert

Reply:
Ah, I see what you mean. Thanks for catching that.   Probably a better way to structure that body of code but that's another matter.

Reverting...